data-products/src/test/scala/org/sunbird/analytics/exhaust/TestOnDemandDruidExhaustJob.scala

```diff
@@ -0,0 +1,713 @@
+        |"school_externalId","organisation_name","program_name","program_externalId","project_id","project_title_editable","project_description","area_of_improvement",
+        |"project_duration","tasks","sub_task","task_evidence","task_remarks","state_slug"],"filters":[{"type":"equals","dimension":"private_program","value":"false"},
+        |{"type":"equals","dimension":"sub_task_deleted_flag","value":"false"},{"type":"equals","dimension":"task_deleted_flag","value":"false"},{"type":"equals",
+        |"dimension":"project_deleted_flag","value":"false"},{"type":"equals","dimension":"program_id","value":"$programId"},{"type":"equals","dimension":"solution_id",
+        |"value":"$solutionId"},{"type":"equals","dimension":"state_slug","value":"$state_slug"}]}}],"labels":{"createdBy":"UUID","designation":"Role",
+        |"state_name":"Declared State","district_name":"District","block_name":"Block","school_name":"School Name","school_externalId":"School ID",
+        |"organisation_name":"Organisation Name","program_name":"Program Name","program_externalId":"Program ID","project_id":"Project ID",
+        |"project_title_editable":"Project Title","project_description":"Project Objective","area_of_improvement":"Category","project_duration":"Project Duration",
+        |"tasks":"Tasks","sub_task":"Sub-Tasks","task_evidence":"Evidence","task_remarks":"Remarks"},"output":[{"type":"csv","metrics":["createdBy","designation",
+        |"state_name","district_name","block_name","school_name","school_externalId","organisation_name","program_name","program_externalId","project_id",
+        |"project_title_editable","project_description","area_of_improvement","project_duration","tasks","sub_task","task_evidence","task_remarks"],
+        |"fileParameters":["id","dims"],"zip":false,"dims":["state_slug","date"],"label":""}]},"store":"local","container":"test-container",
+        |"key":"src/test/resources/ml-druid-data/","format": "csv"},"output":[{"to":"file","params":{"file":"src/test/resources/ml-druid-data-detail"}}],
+        |"parallelization":8,"appName":"ML Druid Data Model"}""".stripMargin
+    val jobConfig = JSONUtils.deserialize[JobConfig](strConfig)
+    implicit val config = jobConfig
+    implicit val conf = spark.sparkContext.hadoopConfiguration
+
+    OnDemandDruidExhaustJob.execute()
+    val postgresQuery = EmbeddedPostgresql.executeQuery("SELECT * FROM job_request WHERE job_id='ml-task-detail-exhaust'")
+    while(postgresQuery.next()) {
+      postgresQuery.getString("status") should be ("SUCCESS")
+      postgresQuery.getString("requested_by") should be ("36b84ff5-2212-4219-bfed-24886969d890")
+      postgresQuery.getString("requested_channel") should be ("ORG_001")
+      postgresQuery.getString("err_message") should be ("")
+      postgresQuery.getString("iteration") should be ("0")
+      postgresQuery.getString("encryption_key") should be ("test@123")
```

Comment:
Also add assertions on download_url field

Reply:
Added the Assertions